KevinHardy-Cooper/VApp
# Added Colors for Settings based on weights

To test:
Running the application with npm start
Enter settings for any social media
Check if the settings state with the highest weights is Red, the lowest weights are Green and weight that is neither the height or lowest are yellow. 

Based on the Current setting the banner changes color
![image](https://user-images.githubusercontent.com/9068012/54870195-af5d0000-4d79-11e9-95f7-64877c6da84b.png)

addresses #127
